docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+
+services:
+
+  consul-agent-1: &consul-agent
+    image: consul:latest
+    networks:
+      - trento
+    command: "agent -retry-join consul-server-bootstrap -client 0.0.0.0"
+
+  consul-agent-2:
+    <<: *consul-agent
+
+  consul-agent-3:
+    <<: *consul-agent
+
+  consul-server-1: &consul-server
+    <<: *consul-agent
+    command: "agent -server -retry-join consul-server-bootstrap -client 0.0.0.0"
+
+  consul-server-2:
+    <<: *consul-server
+
+  consul-server-bootstrap:
+    <<: *consul-agent
+    ports:
+      - "8400:8400"
+      - "8500:8500"
+      - "8600:8600"
+      - "8600:8600/udp"
+    command: "agent -server -bootstrap-expect 3 -ui -client 0.0.0.0"
+
+  trento-webserve:
+    image: trento:latest
```

Comment:
We might actually want to use `build: .`, instead of `image` with something that needs to be created separately with `docker build`, so that `docker-compose up` will do the building for us. :)

Reply:
Totally right, as the docker-compose.yml is just for internal usage it might as well reference the local Dockerfile to build the image. We could maybe switch it to static image names at some point in the future if we decide to upload the container in some registry.